JiraV3 Add The Raw Description To The Incident Context (#38525)
demisto/content: Packs/Jira/ReleaseNotes/3_2_14.md

#### Integrations

##### Atlassian Jira v3

> - Added the **RawDescription** field to the ***fetch-incidents*** context data. This field contains the raw description of the Jira issue.

Reviewer: ```suggestion
- Added the **RawDescription** field to the Jira Issue fields. This field contains the raw description of the Jira issue, without any parsing. 
```

> - Added the **Ticket.RawDescription** field to the context data of the following commands:

Reviewer: ```suggestion
- The field **Ticket.RawDescription** is now available in ***fetch-incidents*** and in the following commands:
```
